session/handler.go

```diff
@@ -53,17 +57,21 @@ func (h *Handler) RegisterAdminRoutes(admin *x.RouterAdmin) {
 		// Redirect to public endpoint
 		admin.Handle(m, RouteWhoami, x.RedirectToPublicRoute(h.r))
 	}
+
+	admin.DELETE(RouteDeleteSession, h.deleteIdentitySessions)
 }
 
 func (h *Handler) RegisterPublicRoutes(public *x.RouterPublic) {
-	// We need to completely ignore the whoami path so that we do not accidentally set
+	// We need to completely ignore the whoami/logout path so that we do not accidentally set
 	// some cookie.
 	h.r.CSRFHandler().IgnorePath(RouteWhoami)
+	h.r.CSRFHandler().IgnoreGlob(RouteIdentity + "/*")
```

Comment:
Please only allow the specific URL added!

Reply:
Done in cf2b30e